compiler/rustc_codegen_ssa/src/mir/mod.rs

```diff
@@ -126,12 +126,7 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
     where
         T: Copy + TypeFoldable<TyCtxt<'tcx>>,
     {
-        debug!("monomorphize: self.instance={:?}", self.instance);
-        self.instance.instantiate_mir_and_normalize_erasing_regions(
-            self.cx.tcx(),
-            self.cx.typing_env(),
-            ty::EarlyBinder::bind(value),
-        )
+        value
```

Comment:
Uh I guess remove this function in a follow-up? 😆

Reply:
Yeah. Leaving this stub in is part of me trying to keep the diff reviewable.